code/modules/mob/living/carbon/complex/gondola.dm

```diff
@@ -19,6 +19,7 @@
 	mob_swap_flags = ALLMOBS
 
 /mob/living/carbon/complex/gondola/New()
+	add_language(LANGUAGE_GALACTIC_COMMON)
```

Comment:
You could also set their universal_speak/universal_understand var to TRUE if you don't want to deal with languages.